frontend/lang/messages/en-US.json

```diff
@@ -985,7 +985,9 @@
       "food-data": "Food Data",
       "example-food-singular": "ex: Onion",
       "example-food-plural": "ex: Onions",
-      "label-overwrite-warning": "This will assign the chosen label to all selected foods and potentially overwrite your existing labels."
+      "label-overwrite-warning": "This will assign the chosen label to all selected foods and potentially overwrite your existing labels.",
+      "on-hand": "On Hand",
```

Comment:
'on-hand' exists elsewhere for the tool.. I decided to add it again in this section as it's specific to the foods, and allows for it to potentially change in future.

Reply:
I don't think this is necessary, I'd rather just change it in the future if we go that route. We have a _ton_ of mixed translations where it's in one category but used for something else

Reply:
Updated